paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -545,15 +544,10 @@ internal class PaymentSheetViewModelTest {
         val selection = PaymentSelection.Saved(PaymentMethodFixtures.CARD_PAYMENT_METHOD)
         viewModel.updateSelection(selection)
 
-        var stripeIntent: StripeIntent? = null
```

Comment:
This change seems unrelated.

Reply:
My last PR updated this test, but @tillh-stripe mentioned that we don't need to use turbine here. I can move it to a different PR if that makes more sense.

Reply:
I see. To be fair, turbine was testing more than what this is testing now. But it would probably make more sense to put the assertions and manipulations in line with the turbine `test` call. 

I'd advocate for doing this in a separate PR.